xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
This is already conditional, the conditionally depends on rouge in code.

```ruby
return snippet.contents unless Rouge
```

I put a hard-dependency in the gemspec since I would assume if you are using RubyGems (a package manager) you would want it to manage your packages and dependencies.